packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -25,3 +25,14 @@ export const getSelectedAccount = (
         ) || null
     );
 };
+
+export const getIsAccountDisconnected = (
+    blockchain: BlockchainNetworks,
+    selectedAccount: SelectedAccountStatus,
+) => {
+    if (selectedAccount.status !== 'loaded') return true;
+    const { symbol } = selectedAccount.network;
+    const chain = blockchain[symbol];
+    if (!chain || chain.connected) return null;
+    return !chain || !chain.connected;
+};
```

Comment:
@marekrjpolak I got this as inspiration from https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/components/wallet/AccountMode/BackendDisconnected.tsx#L61

It looks like after disconnecting (for example disconnecting computer form internet) it takes a while to realize that backend is not connected. I was wondering if you know about better way to get that information?

Reply:
Here the `chain.connected` doesn't work because coinjoin account is using its own non-standard backend instead of default TrezorConnect backends (meaning that you could have `chain.connected = true` even if CJ backend is inaccessible). Consider using `account.status === 'out-of-sync'` instead.

Still, it updates only once in a minute currently, but we could change it if needed.

Reply:
Thanks for the suggestion, I updated it to use `account.status === 'out-of-sync'`  [9720a5b](https://github.com/trezor/trezor-suite/pull/7295/commits/9720a5bafe8e2e578169e4b76882f051f84a1e28)